### PR TITLE
Feature/s2 d item notify order icon

### PR DESCRIPTION
**Pull request by yuying6688**
### Summary
This PR is for PCOM-2278 and PCOM-3155, add order item icon to item order collapsed view. As well as display notify message of item availability checking

### Screenshots
image

image

image

### CODE REVIEW

File: openpos-flow/src/main/java/org/jumpmind/pos/core/ui/data/SellItem.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
     private String optionsLabel;
     private boolean isTender;
     private boolean svgImage;
+    private String iconName;
```
Comment (dankaste) [inline openpos-flow/src/main/java/org/jumpmind/pos/core/ui/data/SellItem.java:39]: Lets call this orderIcon or something more specific

**Review by dankaste — CHANGES_REQUESTED** [review]

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/interfaces/sell-item.interface.ts [hunk 2]
```diff
@@ -32,4 +32,5 @@ export interface ISellItem extends IItem {
     isOrderItem: boolean;
     isTender: boolean;
     svgImage: boolean;
+    orderIcon:string;
```
Comment (kdelorey) [inline openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/interfaces/sell-item.interface.ts:35]: nit: can you add a space between the name and type here.

```suggestion
    orderIcon: string;
```

**Review by kdelorey — DISMISSED** [review]
lgtm.  I just a slight nitpick.

**Review by dankaste — DISMISSED** [review]

File: openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/interfaces/sell-item.interface.ts (hunk 2 above)
Comment (yuying6688) [reply openpos-client-libs/projects/openpos-client-core-lib/src/lib/core/interfaces/sell-item.interface.ts]: Fixed

**Review by kdelorey — APPROVED** [review]

**Review by dankaste — APPROVED** [review]

**Pull request merged**